--- correctOutput/if_hard.s8	2015-01-26 10:32:06.000000000 +0100
+++ testOutput/if_hard.s8	2015-02-05 05:26:21.000000000 +0100
@@ -8,21 +8,5 @@
 .INTEGER: .ascii "%d \000"
 .FLOAT: .ascii "%f \000"
 .NEWLINE: .ascii "\n \000"
-.STRING0: .ascii "A is different from 10"
-.ascii "\000"
-.STRING1: .ascii "B is different from -15"
-.ascii "\000"
-.STRING2: .ascii "B equals -15"
-.ascii "\000"
-.STRING3: .ascii "This never happens..."
-.ascii "\000"
-.STRING4: .ascii "a >0, b > 0"
-.ascii "\000"
-.STRING5: .ascii "a > 0, b <= 0"
-.ascii "\000"
-.STRING6: .ascii "a <= 0, b > 0"
-.ascii "\000"
-.STRING7: .ascii "a <= 0, b <= 0"
-.ascii "\000"
 .globl main
 .align	2
